threescale_api/defaults.py

```diff
@@ -345,9 +358,13 @@ def set(self, item: str, value: Any):
         self.entity[item] = value
 
     def _lazy_load(self, **kwargs) -> 'DefaultResource':
-        if not self._entity:
+        if self._entity is None:
```

Comment:
did pylint blame this? former form looks correct maybe even preferred as "prettier"

Reply:
The problem with 3scale API is its unconsistency. It can return {} and sometimes [] too. I don't remember exact call why I changed this. If you also consider that this check is there only for
```
383     def _invalidate(self):
384         self._entity = None
```
I think the better solution is to check only for `None` and exclude `[], {} and 0` even they are not valid in most of the cases.